src/encoded/static/components/genome_browser.js

```diff
@@ -148,6 +148,8 @@ function mapGenome(inputAssembly) {
  */
 const TrackLabel = ({ file, label, long }) => {
     const biologicalReplicates = file.biological_replicates && file.biological_replicates.join(', ');
+    const splitDataset = file.dataset.split('/');
+    const datasetName = splitDataset[splitDataset.length - 2];
```

Comment:
globals.js has the function `atIdToAccession` that converts an `@id` to an accession. Probably easier to read, as I had to work out what these two lines did at first.

Reply:
Ah, that is very good to know! I would have used that function if I'd realized it existed.